examples/oft-solana/README.md

```diff
@@ -331,14 +331,42 @@ pnpm hardhat lz:oft:solana:setauthority --eid <SOLANA_EID> --mint <TOKEN_MINT> -
 The `OFTStore` is automatically added as a mint authority to the newly created mint authority, and does not need to be
 included in the `--additional-minters` list.
 
-## Common Errors
+## Appendix
 
-### "AnchorError occurred. Error Code: DeclaredProgramIdMismatch. Error Number: 4100. Error Message: The declared program id does not match the actual program id."
+### Transferring ownership
+
+Ownership of OFTs can be transferred via running the wire command after the appropriate changes are made to the LZ Config file (`layerzero.config.ts`). You need to first set the `delegate` value, and then only the `owner` value.
+
+**First, set the `delegate`.**
+
+How to set delegate: https://docs.layerzero.network/v2/developers/evm/create-lz-oapp/configuring-pathways#adding-delegate
+
+Now run
+
+```
+npx hardhat lz:oapp:wire --oapp-config layerzero.config.ts --solana-secret-key <PRIVATE_KEY> --solana-program-id <PROGRAM_ID>
+```
+
+and execute the transactions.
+
+**Then, set the `owner`.**
+
+How to set owner: https://docs.layerzero.network/v2/developers/evm/create-lz-oapp/configuring-pathways#adding-owner
+
+Now, run for another time
+
+```
+npx hardhat lz:oapp:wire --oapp-config layerzero.config.ts --solana-secret-key <PRIVATE_KEY> --solana-program-id <PROGRAM_ID>
```

Comment:
isnt this supposed to be this instead?

>npx hardhat lz:ownable:transfer-ownership --oapp-config layerzero.config.ts

Reply:
both actually work. I did think which was better to tell people to use. But as it stands, `lz:ownable:transfer-ownership` doesn't cover `delegate`, so I thought, why not just keep it simple and ask devs to run the same command twice (in this case, it's `wire`). 

longer term, we should have a single task that takes care of change of `delegate` and `owner`. internally we are toying with the idea of renaming `lz:ownable:transfer-ownership` to something like `lz:access-control:transfer` and have it handle both `delegate` and `owner` changes

do you think think here it's better to just use `lz:ownable:transfer-ownership` ?

Reply:
I didn't think `wire` set the admin though?  Here is the configurator and it only sets the delegate to my knowledge?

https://github.com/LayerZero-Labs/devtools/blob/main/packages/ua-devtools/src/oapp/config.ts#L619

Reply:
@ryandgoulding you're right. I must've gotten things mixed up when I was user-testing. My bad. Will amend this now.

Reply:
@ryandgoulding pushed